Celeste.Mod.mm/Mod/UI/OuiModUpdateList.cs

```diff
@@ -230,5 +274,32 @@ class OuiModUpdateList : Oui, OuiModOptions.ISubmenu {
                 }
             });
         }
+        /// <summary>
+        /// Downloads all automatically updateable mods
+        /// </summary>
+        private void downloadAllMods() {
+            task = new Task(() => {
+                foreach (modUpdateHolder modupdate in updateableMods) {
+                    modupdate.button.Disabled = true;
+                }
+                menu.Focused = false;
+                foreach (modUpdateHolder modupdate in updateableMods) {
+                    doDownloadModUpdate(modupdate.update, modupdate.metadata, modupdate.button);
+                }
+
+                // There should be no more buttons selectable, however lets do this anyway.
+                menu.MoveSelection(1);
+
+                // give the menu control back to the player
+                menu.Focused = true;
```

Comment:
Maybe re-enable buttons for updates that failed?

Edit: okay actually `doDownloadModUpdate` does that already. Maybe re-enable Update All in that case?

Reply:
How I would see it is: the buttons are greyed out, and enabled again when installing all updates is over. In this case, the selection is over the "update all" button.

Reply:
makes a lot of sense, I added it to `downloadAllMods` right next to the code that removes successful updates from the list